elements/pfe-navigation/demo/index.html

```diff
@@ -92,6 +98,22 @@
           border-top: 0px dashed #fff;
           outline: 0;
         }
+
+        pfe-icon.PFElement {
+          --pfe-icon--Color: #fff;
+        }
+      }
+
+      /* @todo: figure out better way to keep custom link text from breaking to multiple lines if at all possible or figure out how to keep the breaking from looking bad */
```

Comment:
Been thinking about this too, I think we're going to have to recommend that they use `&nbsp;` if they don't want a break, or a regular space if they don't mind...

Be curious how you/others feel about that?